Source/Core/Core/ConfigLoaders/IsSettingSaveable.cpp

```diff
@@ -27,9 +27,27 @@ bool IsSettingSaveable(const Config::Location& config_location)
   }
 
   static const auto s_setting_saveable = {
-      &Config::WIIMOTE_1_SOURCE.GetLocation(),  &Config::WIIMOTE_2_SOURCE.GetLocation(),
```

Comment:
Really not sure what's going on here, the linter refused to build this the way it was in master?

Reply:
I believe it's because you added the RA settings later down.

Which brings me to another point: You should not add the RA settings to this list. See 5c3517c31d6b704f3814e55251c9162cf36a74e4.

Reply:
Gotcha, I'll roll that one back then.